docs/domains/Report/services/ReportService/openapi.yml

```
@@ -53,6 +53,65 @@ paths:
                 $ref: '#/components/schemas/ProblemDetails'
         "500":
           description: Internal Server Error
+  /api/Report/Bundle/Manifest:
+    get:
+      tags:
+      - Report
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 8 but found 6

(indentation)